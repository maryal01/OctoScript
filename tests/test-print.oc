print("%d", 10);
print("%.2f", 3.14);
print("test");
print("%d", true);

/* need tests for printing complex types */

