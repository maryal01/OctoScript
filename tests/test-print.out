103.14test1